### PR TITLE
Add napari-metroid recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

---

@jaimergp, @zoccoler do you agree with being added as maintainers?

**Comment by goanpeca** [comment]
@zoccoler, any reason why it is constrained to `python >=3.6,<3.9`? 

Could we relax the upper bound?

Cheers!

**Comment by goanpeca** [comment]
Haven't heard from the author in some time. Moving to review. The author can be added as maintainer in a subsequent PR.

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

**Comment by zoccoler** [comment]
Hi @goanpeca,

Sorry, I did not have the time to check. There was a reason for staying up to 3.8 back then, but I'll check if it is a simple fix.
And sure, I can be maintainer!

**Comment by zoccoler** [comment]
Now it includes python 3.9 ;) @goanpeca

**Comment by jaimergp** [comment]
Hm, line endings are getting in the way :(

File: recipes/napari-metroid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-metroid-{{ version }}.tar.gz
+  sha256: 7d5b1dadbf71f6ce522625422ec09d71dc9f287b89841c5a5ca46c48891e8c7a
+
+build:
+  noarch: python
+  script: 
+    # cheap patch so we don't get line endings issues
+    - sed -i.bak "s/python_requires = =3\.7, <3\.9/python_requires = =3.7, <3.9/" setup.cfg
```
Comment (jaimergp) [inline recipes/napari-metroid/meta.yaml:16]:
```suggestion
    - sed -i.bak "s/python_requires = >=3\.7, <3\.9/python_requires = >=3.7/" setup.cfg
```

**Pull request merged**